emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/write/WriteIopsCalculator.java

```diff
@@ -67,7 +68,7 @@ public WriteIopsCalculator(JobClient jobClient, DynamoDBClient dynamoDBClient, S
   }
 
   public long calculateTargetIops() {
-    double configuredThroughput = Math.floor(getThroughput() * throughputPercent);
+    double configuredThroughput = Math.floor(Double.parseDouble(jobConf.get(DynamoDBConstants.WRITE_THROUGHPUT)) * throughputPercent);
```

Comment:
so, this seems to solve the problem for the Hive's use case (or maybe Spark as well) because you have changed it in `DynamoDBStorageHandler`. however, for those who uses `DynamoDBImport` / `DynamoDBOutputFormat` directly will not be able to get the `configuredThroughput` correctly.

so I would recommend to have the following. Assumed user must set their `dynamodb.throughput.write` as part of the job configuration, and if one didn't, it will fall back to using the table default setup 

```
    double configuredThroughput = Math.floor(Double.parseDouble(
        jobConf.get(DynamoDBConstants.WRITE_THROUGHPUT, String.valueOf(getThroughput()))) * throughputPercent);
```

Reply:
I improved it further by using getThroughput and letting it default to 40k